app/src/main/java/com/pvp/app/ui/screen/survey/Surveys.kt

```diff
@@ -211,11 +245,19 @@ fun FiltersBox(
             modifier = Modifier.fillMaxWidth()
         ) {
             Text(
-                style = TextStyle(fontSize = 18.sp),
                 modifier = Modifier.padding(8.dp),
-                text = title
+                style = TextStyle(fontSize = 18.sp),
+                text = boxTitle
             )
 
+            if (filters.isEmpty()) {
+                Text(
+                    text = if (isSelected == false) "No other $title" else "No $title selected",
```

Comment:
`isSelected` is a non-nullable boolean here.

```kotlin
if (!isSelected) ...
```
I believe there was another component like this one, where boolean was nullable. If you can evade nullability, then evade it. But if nullable was there to indicate some sort of state in which different components are shown, let it be.